src/Perla/Commands.fs

```diff
@@ -786,6 +790,43 @@ Updated: {package.updatedAt.ToShortDateString()}"""
       return 0
     }
 
+  let runRestore () =
+    taskResult {
+      let! fdsConfig = Fs.getPerlaConfig (Path.GetPerlaConfigPath())
+      let! lockFile = Fs.getOrCreateLockFile (Path.GetPerlaConfigPath())
+
+      if lockFile.imports |> Map.isEmpty |> not then
+        return! exn "Import map already exists" |> Error
+
+      let packages =
+        fdsConfig.packages
+        |> Option.defaultValue Map.empty
+
+      let addRuns =
+        packages
+        |> Map.toList
+        |> List.map (fun (k, v) ->
```

Comment:
did a small change here, rather than converting to seq then to list I changed it directly to a list